buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -70,7 +73,11 @@ public void apply(Project project) {
               });
     }
     if (System.getenv().containsKey("METALAVA_BINARY_PATH")) {
-      setupApiInfomrationAnalysis(project, android);
+      try {
+        setupApiInformationAnalysis(project, android);
+      } catch (IOException e) {
```

Comment:
what's the purpose of this? can you remove the catch? also why does this method need to throw IOException?